PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/CosmosSparkBatchRunner.kt

```diff
@@ -50,18 +50,15 @@ class CosmosSparkBatchRunner : SparkBatchJobRunner() {
                                                        submitModel.accountName,
                                                   submitModel.clusterId
                                                           ?: throw ExecutionException("Can't get the Azure Serverless Spark cluster, please sign in and refresh."))
-    ).flatMap { (tenantId, accountName, clusterId) ->
-        val clusterDetail = AzureSparkCosmosClusterManager.getInstance().findCluster(accountName, clusterId)
-                .toBlocking().singleOrDefault(null) ?: throw ExecutionException("Can't get Spark on Cosmos cluster")
-
-        val livyUriOb = submitModel.livyUri
-                ?.let { just(URI.create(it)) }
-                ?: clusterDetail.get().map { it.livyUri }
-
-        livyUriOb.map { livyUri -> CosmosSparkBatchJob(
-                prepareSubmissionParameterWithTransformedGen2Uri(submitModel.submissionParameter),
-                SparkBatchAzureSubmission(tenantId, accountName, clusterId, livyUri),
-                ctrlSubject)
-        }
+    ).flatMap { (tenantId, accountName, clusterId) -> AzureSparkCosmosClusterManager.getInstance()
+            .findCluster(accountName, clusterId)
```

Comment:
Could you help test the following scenario with the current code?
1. Select a cluster in configuration, save the configuration, close the configuration dialog.  
2.De-select the subscription you choose last time. Run the job. Will we warn user that the compute doesn't exist?

Reply:
Thanks your suggestion! It's no problem like this:
![image](https://user-images.githubusercontent.com/7385667/74911712-22428100-53f8-11ea-98ee-44d1c1c905ef.png)